src/ensembl/src/content/app/entity-viewer/gene-view/components/transcripts-table/main-content/MainContent.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+import styles from './MainContent.scss';
+import tableStyles from '../TranscriptsTable.scss';
+
+const MainContent = () => {
+  return (
+    <div className={`${tableStyles.tableMiddle} ${styles.mainContent}`}>
+      <div className={tableStyles.tableHeader}>
+        <div className={tableStyles.tableRow}>
+          <span className={tableStyles.tableCell}>Biotype</span>
+          <span className={tableStyles.tableCell}>Spliced RNA length</span>
+          <span className={tableStyles.tableCell}>Coding exons</span>
+          <span className={tableStyles.tableCell}>Protein</span>
+          <span className={tableStyles.tableCell}>Transcript name</span>
+          <span className={tableStyles.tableCell}>TSL</span>
+        </div>
+      </div>
+      <div className={tableStyles.tableBody}>
+        <div className={tableStyles.tableRow}>
+          <span className={tableStyles.tableCell}>Protein coding</span>
+          <span className={tableStyles.tableCell}>11,986</span>
+          <span className={tableStyles.tableCell}>26 of 27</span>
+          <span className={tableStyles.tableCell}>3418aa</span>
```

Comment:
notice that the design has a non-breaking space between the number and the aa (although I don't like this aa thing here at all :disappointed:  )

Reply:
:heavy_check_mark:

Reply:
I checked with Andrea about this. She said that we need to use `aa` as well as `bp` in the table. So updated accordingly.